docker-root/usr/local/bin/start.sh

```diff
@@ -6,19 +6,19 @@ detect-iptables.sh
 [ -n "$CHECK_SYSTEM_ONLY" ] && exit
 
 cp /etc/danted.conf.sample /run/danted.conf
-externals=""
-for iface in $({ ip -f inet -o addr; ip -f inet6 -o addr; } | sed -E 's/^[0-9]+: ([^ ]+) .*/\1/'); do
-	externals="${externals}external: $iface\\n"
+internals=""
+for iface in $(ip -o addr | sed -E 's/^[0-9]+: ([^ ]+) .*/\1/' | sort | uniq | grep -v "lo\|sit\|vir"); do
+	internals="${internals}internal: $iface port = 1080\\n"
 done
-sed s/^#external-lines/"$externals"/ -i /run/danted.conf
+sed /^internal/c"$internals" -i /run/danted.conf
 # 在虚拟网络设备 tun0 打开时运行 danted 代理服务器
 [ -n "$NODANTED" ] || (while true
 do
 sleep 5
 [ -d /sys/class/net/tun0 ] && {
 	chmod a+w /tmp
 	open_port 1080
-	su daemon -s /usr/sbin/danted -f /run/danted.conf
+	/usr/sbin/danted -f /run/danted.conf
```

Comment:
先前将 danted 放在 daemon 用户下运行是为了消除 danted 用 root 用户运行时的警告（另外理论上确实也更安全），此处的改动是否有什么原因？
```suggestion
	su daemon -s /usr/sbin/danted -f /run/danted.conf
```

Reply:
这里可能不能依赖daemon用户，或许有些发行系都没有daemon用户
PR中添加了socks用户，danted运行在socks用户下
Dockerfile 
`RUN groupadd -r socks && useradd -r -g socks socks`
danted.conf
`user.notprivileged: socks`
PS:danted.conf套用了官方的完整最小模板，和原先的差异大概就是日志了，其他应该没有什么改变
PR的代码已打包测试过